service/history/config/config.go

```diff
@@ -323,10 +323,13 @@ type Config struct {
 	LargeShardHistorySizeMetricThreshold  dynamicconfig.IntPropertyFn
 	LargeShardHistoryEventMetricThreshold dynamicconfig.IntPropertyFn
 	LargeShardHistoryBlobMetricThreshold  dynamicconfig.IntPropertyFn
+
+	// Hostname addition
+	HostName string
```

Comment:
... addition of what?

Reply:
Add description of HostName in newest diff